src/utils/fileUtils.test.tsx

```
@@ -107,10 +107,57 @@ describe('fetchFileAndExtractText', () => {
 
+    const originalSplit = String.prototype.split; // Save before spying
+
+    const splitSpy = vi
+      .spyOn(String.prototype, 'split')
+      .mockImplementation(function (
+        this: string,
+        separator: string | RegExp | { [Symbol.split](string: string, limit?: number): string[] },
+        limit?: number
+      ) {
+        if (this !== slashUrl) {
+          return originalSplit.call(this, separator as any, limit); // Use saved original
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)